src/app/data-planes/components/__snapshots__/DataPlaneDetails.spec.ts.snap

```diff
@@ -281,7 +281,7 @@ exports[`DataPlaneDetails matches snapshot 1`] = `
                               
                               <a
                                 class=""
-                                href="/mesh/default/services/backend"
+                                href="/mesh/default/service/backend"
```

Comment:
All of the snapshot changes are as a result of having detail pages at `services` as opposed to `services`